src/strings/states.js

```diff
@@ -0,0 +1,55 @@
+const STATE_OPTIONS = [
```

Comment:
It's done, but in the future we can use a package to grab US states https://www.npmjs.com/package/usa-states.  Just one less thing someone has to think about when interacting with the project.

Reply:
The benefit seems marginal because the states are not likely to change and we will probably have advance notice if we do. With a third party we'd have to manage formatting and rely on their accuracy and maintenance.  

I used this gist https://gist.github.com/mshafrir/2646763 and multiple cursors so it was trivial to implement and likely never needs to be touched/ installed/ formatted again unless Grommet has an api change.